project/mobile/steps/navigation_steps/navigation_steps.py

```diff
@@ -1,15 +1,15 @@
-from common.facade import logger
-
 __all__ = ['navigate_to_main_page']
 
+from common.facade import logger
+
 from project.mobile.src.confirmations.location_permission import LocationPermissionAndroid
-from project.mobile.steps.pages.setup_notification import SetupNotificationSteps
-from project.mobile.steps.pages.setup_location import SetupLocationSteps
-from project.mobile.steps.pages.weather import WeatherSteps
+from project.mobile.steps.page_steps.pages.setup_notification import SetupNotificationSteps
+from project.mobile.steps.page_steps.pages.setup_location import SetupLocationSteps
+from project.mobile.steps.page_steps.pages.weather import WeatherSteps
 
 
 def navigate_to_main_page():
-    logger.log_step("Navigate to main Weather Page", precondition=True)
+    logger.log_title("Navigate to main Weather Page")
```

Comment:
location_permission.deny() can't find element Deny on the screen

Reply:
which version of android you use for tests? could you share screen with fail result?